mappings/net/minecraft/container/MerchantContainer.mapping

```diff
@@ -1,22 +1,31 @@
 CLASS ayy net/minecraft/container/MerchantContainer
-	FIELD c villager Lbgd;
-	FIELD d villagerInventory Layx;
+	FIELD c trader Lbgd;
+	FIELD d traderInventory Layx;
```

Comment:
Why do we have `MerchantContainer` but `traderInventory`? I think we should either use "merchant" or "trader". Do we know which one is the Mojang name?

Reply:
mojabg calls the gui merchant container.
btw, where does mojang call trader?

Reply:
Weird, the ID for the trader is `wandering_trader`, but the container is `merchant` rather than `trader`...

Reply:
villagers are merchants too. The UI is a merchant UI and the entity is a trader. It makes sense to me because merchants don't necessarily have to be traders, even if that's all vanilla ones are at the moment.

Reply:
so `merchant` for all recipes, ui, etc.?